runtime/init.go

```diff
@@ -63,9 +63,20 @@ func (d *Default) init(cluster *v1.Cluster) error {
 		return err
 	}
 
+	if err := d.GetKubectlAndKubeconfig(); err != nil {
```

Comment:
if master is changed. only at init step is not complete. I think the copy config need at all node change step.

Reply:
If changed, User copy it by himself, If you copy it every time, it's inefficient.